### PR TITLE
Fullscreen resource calendar

### PR DESCRIPTION
**Pull request by AndrewVos**
screen shot 2016-11-17 at 14 16 21

### CODE REVIEW

**Comment by benbarnett** [comment]
Looking a little crazy in IE11, although it works functionally:

screen shot 2016-11-15 at 13 18 55

File: app/views/resource_calendars/show.html.erb [hunk 1]
```diff
@@ -5,44 +5,46 @@
...
-    select_tag(
-      :filter_guiders,
-      grouped_options_for_select(guider_options),
-      multiple: 'multiple',
-      class: 't-users',
-      data: { module: 'advanced-select' }
-    )
-  %>
-</div>
+<div class='fullscreen-container'>
```
Comment (Guntrisoft) [inline app/views/resource_calendars/show.html.erb]: double quotes

**Comment by AndrewVos** [comment]
@benbarnett Is the IE thing because we don't set a background colour in the container maybe?

**Comment by benlovell** [comment]
:shipit:

**Pull request merged**